plotly/tools.py

```diff
@@ -1443,6 +1449,333 @@ class FigureFactory(object):
+
+        # Make data for plot
+        u=np.linspace(0, 2*np.pi, 20)
+        v=np.linspace(0, 2*np.pi, 20)
+        u,v=np.meshgrid(u,v)
+        u=u.flatten()
+        v=v.flatten()
+
+        x = (3 + (np.cos(v)))*np.cos(u)
+        y = (3 + (np.cos(v)))*np.sin(u)
+        z = np.sin(v)
+
+        points2D = np.vstack([u,v]).T
+        tri = Delaunay(points2D)
+        simplices = tri.simplices
+
+        # Create a figure
+        fig1 = FF.create_trisurf(x=x, y=y, z=z,
+                                 colormap="Portland",
+                                 simplices=simplices)
+        # Plot the data
+        py.iplot(fig1, filename='Trisurf Plot')
+        ```
+
+        Example 3: Mobius Band
+        ```
+        # Necessary Imports for Trisurf
+        import numpy as np
+        from scipy.spatial import Delaunay
+
+        import plotly.plotly as py
+        from plotly.tools import FigureFactory as FF
+        from plotly.graph_objs import graph_objs
+
+        # Make data for plot
+        u=np.linspace(0, 2*np.pi, 24)
+        v=np.linspace(-1, 1, 8)
+        u,v=np.meshgrid(u,v)
+        u=u.flatten()
+        v=v.flatten()
+
+        tp = 1 + 0.5*v*np.cos(u/2.)
+        x=tp*np.cos(u)
+        y=tp*np.sin(u)
+        z=0.5*v*np.sin(u/2.)
+
+        points2D = np.vstack([u,v]).T
+        tri = Delaunay(points2D)
+        simplices = tri.simplices
+
+        # Create a figure
+        fig1 = FF.create_trisurf(x=x, y=y, z=z,
+                                 colormap=[(0.2, 0.4, 0.6),(1, 1, 1)],
+                                 simplices=simplices)
+        # Plot the data
+        py.iplot(fig1, filename='Trisurf Plot')
+        ```
+        """
+        from plotly.graph_objs import graph_objs
+        plotly_scales = {'Greys': ['rgb(0,0,0)', 'rgb(255,255,255)'],
+                         'YlGnBu': ['rgb(8,29,88)', 'rgb(255,255,217)'],
+                         'Greens': ['rgb(0,68,27)', 'rgb(247,252,245)'],
+                         'YlOrRd': ['rgb(128,0,38)', 'rgb(255,255,204)'],
+                         'Bluered': ['rgb(0,0,255)', 'rgb(255,0,0)'],
+                         'RdBu': ['rgb(5,10,172)', 'rgb(178,10,28)'],
+                         'Reds': ['rgb(220,220,220)', 'rgb(178,10,28)'],
+                         'Blues': ['rgb(5,10,172)', 'rgb(220,220,220)'],
+                         'Picnic': ['rgb(0,0,255)', 'rgb(255,0,0)'],
+                         'Rainbow': ['rgb(150,0,90)', 'rgb(255,0,0)'],
+                         'Portland': ['rgb(12,51,131)', 'rgb(217,30,30)'],
+                         'Jet': ['rgb(0,0,131)', 'rgb(128,0,0)'],
+                         'Hot': ['rgb(0,0,0)', 'rgb(255,255,255)'],
+                         'Blackbody': ['rgb(0,0,0)', 'rgb(160,200,255)'],
+                         'Earth': ['rgb(0,0,130)', 'rgb(255,255,255)'],
+                         'Electric': ['rgb(0,0,0)', 'rgb(255,250,220)'],
+                         'Viridis': ['rgb(68,1,84)', 'rgb(253,231,37)']}
```

Comment:
> going to a 9 color scale instead of 2

Do we need to have a 9 color scale? My stuff so far (scatterplotmatrix and trisurf) only requires and uses 2 color lists and just computes the distance between the two colors for heatmapping numeric values, etc. Is it computationally more efficient to go with a larger number, or do other functions use a larger number of colors?